packages/core/src/components/Section/Section.tsx

```diff
@@ -0,0 +1,126 @@
+import { HvBaseProps } from "@core/types/generic";
+import { ExtractNames } from "@core/utils/classes";
+import {
+  HvActionGeneric,
+  HvActionsGenericProps,
+} from "@core/components/ActionsGeneric";
+import { HvButton, HvButtonProps } from "@core/components/Button";
+import { HvDropDownMenu } from "@core/components/DropDownMenu";
+import { HvTypography } from "@core/components/Typography";
+import { useDefaultProps } from "@core/hooks/useDefaultProps";
+import { useControlled } from "@core/hooks/useControlled";
+import { Down, Up } from "@hitachivantara/uikit-react-icons";
+
+import { staticClasses, useClasses } from "./Section.styles";
+
+export { staticClasses as sectionClasses };
+
+export type HvSectionClasses = ExtractNames<typeof useClasses>;
+
+export interface HvSectionProps
+  extends Omit<HvBaseProps<HTMLDivElement>, "title"> {
+  /** The title of the section */
+  title: React.ReactNode;
+  /** Whether or not the section is expandable.  */
+  expandable?: boolean;
+  /** Whether the section is open or not, if this property is defined the accordion must be fully controlled. */
+  expanded?: boolean;
+  /** When uncontrolled, defines the initial expanded state. */
+  defaultExpanded?: boolean;
+  /** Section actions */
+  actions?: HvActionGeneric[];
+  /** Section action callback */
+  actionCallback?: HvActionsGenericProps["actionsCallback"];
+  /** Section maximum number of actions visible */
+  maxVisibleActions?: number;
+  /** Variant for the visible actions button */
+  actionsVariant?: HvButtonProps["variant"];
+  /** Section onExpand callback */
+  onToggle?: (id: string | undefined, open: boolean) => void;
+  /** A Jss Object used to override or extend the styles applied to the empty state component. */
+  classes?: HvSectionClasses;
+}
+
+/**
+ * Sections allow grouping information on a page under the same topic.
+ */
+export const HvSection = (props: HvSectionProps) => {
+  const {
+    id,
+    classes: classesProp,
+    className,
+    title,
+    expandable,
+    expanded,
+    defaultExpanded = false,
+    actions,
+    actionCallback,
+    maxVisibleActions = 1,
+    actionsVariant = "secondarySubtle",
+    onToggle,
+    children,
+    ...others
+  } = useDefaultProps("HvSection", props);
+
+  const { classes, cx } = useClasses(classesProp);
+
+  const [isOpen, setIsOpen] = useControlled(expanded, Boolean(defaultExpanded));
+
+  const actsVisible = actions?.slice(0, maxVisibleActions);
+  const actsDropdown = actions?.slice(maxVisibleActions);
+
+  const showContent = expandable ? !!isOpen : true;
+
+  return (
+    <div className={cx(classes.root, className)} {...others}>
```

Comment:
We are missing the `id` that we extracted:

```suggestion
    <div id={id} className={cx(classes.root, className)} {...others}>
```